common/transform/src/main/java/ai/h2o/mojos/deploy/common/transform/MojoScorer.java

```diff
@@ -130,13 +143,10 @@ private static MojoPipeline loadMojoPipelineFromFile() {
     }
     try {
       MojoPipeline mojoPipeline = MojoPipeline.loadFrom(mojoFile.getPath());
```

Comment:
nit: Should we keep the other log entries here too? Not pushing, just asking.

Reply:
I believe I moved the log line into the `ModelsApiController.java` so its not gone... just moved. 

but this is the better location. less code